pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -143,11 +143,20 @@ func (rc *reconciler) isMultiAttachForbidden(volumeSpec *volume.Spec) bool {
 			volumeSpec.Volume.Cinder != nil {
 			return true
 		}
+		// Check for volume types which do not fail when trying to multi-attach
+		if volumeSpec.Volume.VsphereVolume != nil {
+			return false
+		}
 	}
 
 	// Only if this volume is a persistent volume, we have reliable information on wether it's allowed or not to
 	// multi-attach. We trust in the individual volume implementations to not allow unsupported access modes
 	if volumeSpec.PersistentVolume != nil {
+		// Check for persistent volume types which do not fail when trying to multi-attach
+		if volumeSpec.PersistentVolume.Spec.VsphereVolume != nil {
+			return false
+		}
+
 		if len(volumeSpec.PersistentVolume.Spec.AccessModes) == 0 {
```

Comment:
Discussed offline. Ok with leaving this as is for now, and modifying it is also an issue for other storage plugins.